common/actions/initialize/main.js

```diff
@@ -1,16 +1,20 @@
-const utils = require('../../../utils');
+const { getInput, execStep, setFailed } = require('../../../utils');
 
 try{
-    const ghToken = utils.getInput('gh-token');
+    const npmNexusAuth = getInput('nexus-token');
 
-    utils.execStep(
-        `npm config set //npm.pkg.github.com/:_authToken ${ghToken}`,
+    execStep(
+        [
+            `npm config set //repositories.mrf.io/nexus/repository/npm-internal/:_authToken ${npmNexusAuth}`,
+            'npm config set registry https://repositories.mrf.io/nexus/repository/npm-internal/',
```

Comment:
I'm not sure we should set the registry here. Currently, all providers configure the registry in the publishConfig of the package.json, so it won't even be used. Maybe we could remove it from the package.jsons and centralize it here, WDYT?

Reply:
i would use publishConfig together with auth part globally defined

Reply:
```suggestion
```